Makefile

```diff
@@ -3,17 +3,17 @@ NAME := mysql-db-admin
 .PHONY: tools build
 
 tools:
-	cd ~ && go get -u github.com/aws/aws-lambda-go/cmd/build-lambda-zip && cd -
+	go install github.com/aws/aws-lambda-go/cmd/build-lambda-zip@latest
 
 build:
 	mkdir -p ./aws/tf/files
 	GOOS=linux GOARCH=amd64 go build -o ./aws/tf/files/mysql-db-admin ./aws/
+	rm -rf ./gcp/tf/files/ && mkdir -p ./gcp/tf/files/

```

Comment:
ROFL, I see what you mean. It does look like you were coding while on crack here.